scripts/ci/rerun-integration-tests.sh

```diff
@@ -12,10 +11,16 @@ PROJECT_ROOT="$( cd "$( echo "${SCRIPT_PATH}" | sed s+/scripts/ci++)" && pwd )"
 
 source "$SCRIPT_PATH/../utils/test"
 source "$SCRIPT_PATH/../utils/integration-tests"
-
+set +e
```

Comment:
IIRC you can catch exit code from `find_files_with_debug_directives` by directly executing if condition:
```suggestion
if [ find_files_with_debug_directives "$PROJECT_ROOT/$TESTS_FOLDER" ]; then
  echo "No files found to run, aborting"
  exit 0
fi
```

Reply:
also, as `set -e` is removed at the top of this file, this switch isnt necessary anymore, isnt it? 🤔

Reply:
I see, I inlined the subshell and that also worked without the `+e`, so I'm using that now